packages/dds/signal/package.json

```diff
@@ -0,0 +1,157 @@
+{
+	"name": "@fluidframework/signal",
```

Comment:
i think we should make this @fluidprivate scope for now, and set "private": true, so it doesn't publish externally.